include/utils/parameters.h

```diff
@@ -301,6 +301,22 @@ void Parameters::Parameter<T>::print (std::ostream & os) const
   print_helper(os, static_cast<const T *>(&_value));
 }
 
+template <>
+inline
+void Parameters::Parameter<char>::print (std::ostream & os) const
+{
+  // Specialization so that we don't print out unprintable characters
+  os << (int)_value;
```

Comment:
Hmm... not `print_helper(os, static_cast<const int *>(&_value)`.  I think we should try and keep it consistent, just in case `print_helper` is ever updated to do something else some day...